datafusion/core/src/physical_planner.rs

```diff
@@ -2413,9 +2411,6 @@ mod tests {
             "SUM(aggregate_test_100.c2)",
             final_hash_agg.schema().field(1).name()
         );
-        // we need access to the input to the partial aggregate so that other projects can
-        // implement serde
-        assert_eq!("c2", final_hash_agg.input_schema().field(1).name());
```

Comment:
This test is from a time when DataFusion did not have serde built in. I no longer think this is required. See  https://github.com/apache/arrow-datafusion/pull/7741#discussion_r1346319872 for details